tests/tx.test.js

```diff
@@ -1,149 +1,371 @@
+      exitInfo.merkleProof.siblings,
+      true,
+      accounts[0].signer
+    )
+    await withdrawTxData.wait()
 
-    expect(nonInstantWithdrawTxData).toMatchObject({
-      from: accountEthereumAddress,
-      to: CONTRACT_ADDRESSES[ContractNames.Hermez],
-      value: BigNumber.from(0)
-    })
+    const newBalance = Scalar.e(await provider.getBalance(getEthereumAddress(hermezEthAddress)))
 
-    await advanceTime()
+    expect(Scalar.gt(newBalance, oldBalance)).toBe(true)
+  })
 
-    const delayedWithdrawTxData = await Tx.delayedWithdraw(account.hermezEthereumAddress, tokens[1])
+  test('Force exit', async () => {
+    // setup amounts
+    const forceExitAmount = getTokenAmountBigInt('0.29', tokenEth.decimals)
+    const compressedForceExitAmount = HermezCompressedAmount.compressAmount(forceExitAmount)
 
-    expect(delayedWithdrawTxData).toMatchObject({
-      from: accountEthereumAddress,
-      to: CONTRACT_ADDRESSES[ContractNames.WithdrawalDelayer],
-      value: BigNumber.from(0)
-    })
-  })
-})
+    const forceExitData = await Tx.forceExit(
+      compressedForceExitAmount,
+      accounts[1].index,
+      tokenEth,
+      accounts[1].signer
+    )
+    const forceExitReceipt = await forceExitData.wait()
+    const txId = getL1TxIdFromReceipt(forceExitReceipt)
+    await assertTxForged(txId)
 
-test('#sendL2Transaction', async () => {
-  jest.mock('axios')
-
-  const txId = '0x000000000000000007000300'
-  const bjj = ''
-  const tx = {
-    id: txId,
-    bJJ: bjj,
-    nonce: 2
-  }
-  axios.post = jest.fn().mockResolvedValue({
-    status: 200,
-    data: txId
+    // update and assert balances
+    accounts[1].expectedBalance = Scalar.sub(accounts[1].expectedBalance, forceExitAmount)
+    await assertBalances(accounts, tokenEth)
   })
-  TransactionPool.initializeTransactionPool()
 
-  const txResult = await Tx.sendL2Transaction(tx, bjj)
-  expect(txResult).toEqual({
-    status: 200,
-    id: txId,
-    nonce: 2
+  test('Withdrawal from force exit', async () => {
+    const hermezEthAddress = accounts[1].hermezWallet.hermezEthereumAddress
+    const exitInfoAll = await CoordinatorAPI.getExits(hermezEthAddress, true)
+    const exitInfo = exitInfoAll.exits[0]
+
+    const oldBalance = Scalar.e(await provider.getBalance(getEthereumAddress(hermezEthAddress)))
+
+    const withdrawTxData = await Tx.withdraw(
+      exitInfo.balance,
+      accounts[1].index,
+      tokenEth,
+      accounts[1].hermezWallet.publicKeyCompressedHex,
+      exitInfo.batchNum,
+      exitInfo.merkleProof.siblings,
+      true,
+      accounts[1].signer
+    )
+    await withdrawTxData.wait()
+
+    const newBalance = Scalar.e(await provider.getBalance(getEthereumAddress(hermezEthAddress)))
+
+    expect(Scalar.gt(newBalance, oldBalance)).toEqual(true)
   })
 
-  const transactionPool = JSON.parse(TransactionPool._storage.getItem(TRANSACTION_POOL_KEY))
-  expect(transactionPool[bjj]).toEqual([tx])
+  test.skip('#sendL2Transaction mock', async () => {
```

Comment:
Why is it skipped?

Reply:
This is already tested in https://github.com/hermeznetwork/hermezjs/blob/feature/transferToBJJ/tests/tx.test.js#L216
It is also unit testing and may not fit in this `tx.test.js`.
I would keep like this until it is decided how to organize tests